02_configure_host.sh

```diff
@@ -13,6 +13,18 @@ source lib/releases.sh
 # shellcheck disable=SC1091
 source lib/image_prepull.sh
 
+# (workaround) disable tdp_mmu to avoid
```

Comment:
So earlier this was under 01, so moving this to 02 shouldn't make any difference, since we backed down from skipping 01 and like @adilGhaffarDev said, 01 is run always again. But this also moves it to top level, so it is no longer run only on Ubuntu 22, but Ubuntu 18, Ubuntu 20 and Centos. 

Since Ubuntu 22 is failing as well, I'm not sure why this would help... If someone wants to LGTM it, I don't object tho.

Reply:
As we are getting this issue in both ubuntu and centos, this part of the code is valid for both.

Reply:
yeah, I have forgot we again run 01, but yeah I would like to be sure this is being applied, the error is just too similar to this old issue (doesn't makes sense to me either....)